.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,63 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: 'BUG REPORT: '
+labels: ':type: bug'
+assignees: ''
+
+---
+
+*NOTE: This issue template is being created and may not be complete.*
+
+*NOTE: Please only submit bug reports to this repository for issues with the
+contents of the repository itself. General bug reports for Rocky Linux should
+be submitted to the [Rocky Linux Bug Tracker](https://bugs.rockylinux.org).*
+
+*NOTE: Please remove / add sections to this bug report template as you feel
+are necessary to completely describe your issue.*
+
+---
+
+**Describe the bug**
+
+A clear and concise description of what the bug is.
+
+---
+
+**To Reproduce**
+
+Steps to reproduce the behavior:
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+---
+
+**Expected behavior**
+
+A clear and concise description of what you expected to happen.
+
+---
+
+**Screenshots**
+
+If applicable, add screenshots to help explain your problem.
+
+---
+
+**System / Environment (please complete the following information):**
+
+- OS: [e.g. Rocky Linux, CentOS, ...]
```

Comment:
My only question here is shouldn't we assume that whoever is submitting this is running Rocky? Otherwise, this looks good.

Reply:
I'm imagining the situation where we are running a test to verify bug-for-bug behavior. The test may fail in the other operating system but not in Rocky. I want to be sure we don't follow the wrong path attempting to replicate in the wrong OS in that case. Explicitly reminding ISSUE author to include the OS should help prevent that.